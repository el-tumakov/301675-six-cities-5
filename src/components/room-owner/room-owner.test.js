import React from "react";
import renderer from "react-test-renderer";
import RoomOwner from "./room-owner";


const DESCRIPTION = `Discover daily local life in city center, friendly neighborhood, clandestine casino, karaoke, old-style artisans, art gallery and artist studio downstairs.`;
const OWNER = {
  id: 25,
  avatar: `img/avatar-angelina.jpg`,
  name: `Angelina`,
  super: true
};


it(`Should RoomOwner render correctly`, () => {
  const tree = renderer
      .create(
          <RoomOwner
            description={DESCRIPTION}
            owner={OWNER}
          />
      ).toJSON();

  expect(tree).toMatchSnapshot();
});
