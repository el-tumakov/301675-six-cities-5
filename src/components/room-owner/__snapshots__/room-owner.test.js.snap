// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should RoomOwner render correctly 1`] = `
<div
  className="property__host"
>
  <h2
    className="property__host-title"
  >
    Meet the host
  </h2>
  <div
    className="property__host-user user"
  >
    <div
      className="property__avatar-wrapper property__avatar-wrapper--pro user__avatar-wrapper"
    >
      <img
        alt="Host avatar"
        className="property__avatar user__avatar"
        height="74"
        src="img/avatar-angelina.jpg"
        width="74"
      />
    </div>
    <span
      className="property__user-name"
    >
      Angelina
    </span>
  </div>
  <div
    className="property__description"
  >
    <p
      className="property__text"
    >
      Discover daily local life in city center, friendly neighborhood, clandestine casino, karaoke, old-style artisans, art gallery and artist studio downstairs.
    </p>
  </div>
</div>
`;
